client/web/src/enterprise/own/admin-ui/query.ts

```diff
@@ -0,0 +1,30 @@
+import { gql } from '@sourcegraph/http-client'
+
+export const OwnSignalFragment = gql`
```

Comment:
I guess it still should be in a snake case
```suggestion
export const OWN_SIGNAL_FRAGMENT = gql`
```